vitest.config.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
+
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    typecheck: {
+      tsconfig: "./tests/tsconfig.json",
```

Comment:
Not a terribly complicated config but it was required to make sure the types were ambiently there.